node/src/GlideClusterClient.ts

```diff
@@ -943,6 +999,8 @@ export class GlideClusterClient extends BaseClient {
     /**
      * Returns information about the functions and libraries.
      *
+     * The command will be routed to a random node, unless `route` is provided.
```

Comment:
Add `@remakrs` and move it right after `@see`

Reply:
Other commands have the same - I'll update all of them in bulk later

Reply:
I added a task reminder to do that when we will work on docs.